site/src/components/select-anatomy.jsx

```diff
@@ -7,13 +7,15 @@ const SelectAnatomy = () => {
       <Host name="select">
         <Slot name="button">
           <Part name="button type=selectlist">
-            <Part name="selectedvalue">Currently selected value</Part>
+            <Part name="selectedoption">Currently selected option</Part>
+            <Part name="marker"></Part>
```

Comment:
Not worth blocking on - but do you think we should represent these as pseudos? eg: `::marker`?

Reply:
Yes the explainer itself mentions some of the pseudos. Hopefully the new anatomy schema will allow us to make clear what's a pseudo and what's an element proper.